packages/models-library/src/models_library/rabbitmq_messages.py

```diff
@@ -54,6 +55,11 @@ class ProgressType(StrAutoEnum):
     SERVICE_STATE_PUSHING = auto()
     SERVICE_OUTPUTS_PUSHING = auto()
 
+    PROJECT_CLOSING = auto()
+
+
+_UNUSED_NODE_ID: Final[UUID] = UUID(int=0)
```

Comment:
well that is kind of a hack. it is a ```NodeMessageBase``` as with Node in the name... so it is a bit weird to have an optional Node in there now no?

Reply:
Since we discussed and I agreed.. let's come up with some ideas to make it maybe not too confusing.
We could have someting like this:
```python
class ProgressRabbitMessageNode:
    ...
class ProgressRabbitMessageProject:
    ...
```

Also please bear in mind, that I was wondering if I should not make this dataclass-based instead since the parsing is probably not very useful there (but that would be for another PR).
@pcrespov, if you have some other ideas.. please shoot

Reply:
@sanderegg sorry, i have just seen your comment:

IMO a nullable state by default means that is is undefined. There are some cases in which it can be associated a new meaning and then in that case it make sense to create a extra "sentinel" like you do. The disadvantage with the latter is that you need to be aware that the field has a non-default "undefined" state. For instance , assume we refer to the field ``foo``. Then notice the difference:
```python
if model.foo is None:
   print("is undefined")
```
vs
```python
from module import _undefined_foo

if model.foo is _undefined_foo:
   print("is undefined")
```